src/test/java/ToDoTest.java

```diff
@@ -0,0 +1,22 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ToDoTest {
+    @Test
+    public void getTaskDescription () {
+        assertEquals("Borrow book", new ToDo("Borrow book").getDescription());
+    }
+
+    @Test
+    public void getDoneStatus() {
+        assertEquals(false, new ToDo("Borrow book").getDoneStatus());
+    }
+
+    @Test
+    public void getDoneStatusAfterMarkAsCompleted() {
```

Comment:
Consider following this naming convention for test methods:
"Underscores may be used in test method names using the following three part format <code>featureUnderTest_testScenario_expectedBehavior()</code>"

Reply:
Thanks for pointing this out! Will update the name to follow this style!